lib/locations/locations.go

```diff
@@ -128,6 +134,43 @@ func expandLocations() error {
 	return nil
 }
 
+// ListExpandedPaths returns a machine-readable mapping of the currently
+// configured locations.  Unexpanded entries are filtered out.
```

Comment:
What's the reason to filter those? Plus that shouldn't ever happen, right? (expansion happens at init time)

Reply:
Mostly the `${timestamp}` stuff which is expanded when creating such a file.  And I wanted to make sure whatever happens in other code, the returned paths are always real.  So yes, with the current usage pattern this should not actually filter out anything.

Reply:
Given it shouldn't happen, returning regardless of placeholders seems safer (and simpler). Then at least it's obvious that and what is wrong, while just a missing path is easy to miss.

Reply:
Okay, but I'll keep filtering the `${timestamp}` stuff, as those aren't really paths at any given time.

Reply:
I don't care much honestly, but I don't see why? These paths are informational - and the panic-log path template is still useful information and a `${timestamp}` seems self-explantory enough to understand what it means to me.

Reply:
I guess it made more sense when the results of that API call were displayed as a whole in the GUI dialog.  Now that we pick explicit values in the GUI code with more elaborate descriptions, it's fine to have them in the API response and just not include them in the dialog (because non-tech-savvy users maybe won't find it self-explanatory).